### PR TITLE
Test AWS 2 DynamoDB properly

### PR DESCRIPTION
**Pull request by ppalaga**
cc @llowinge

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: integration-tests-aws2/aws2-ddb/src/test/java/org/apache/camel/quarkus/component/aws2/ddb/it/Aws2DdbTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+@QuarkusTest
+@QuarkusTestResource(Aws2TestResource.class)
+@EnabledIfEnvironmentVariable(named = "AWS_ACCESS_KEY", matches = "[a-zA-Z0-9]+") // TODO
+                                                                                  // https://github.com/apache/camel-quarkus/issues/2216
+class Aws2DdbTest {
+
+    private static final Logger LOG = Logger.getLogger(Aws2DdbTest.class);
+
+    @Test
+    public void sqs() {
```
Comment (aldettinger) [inline integration-tests-aws2/aws2-ddb/src/test/java/org/apache/camel/quarkus/component/aws2/ddb/it/Aws2DdbTest.java:44]: sqs ? maybe it's on purpose, I just want to raise attention.

File: integration-tests-aws2/aws2-ddb/src/test/java/org/apache/camel/quarkus/component/aws2/ddb/it/Aws2DdbTestEnvCustomizer.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    @Override
+    public Service[] localstackServices() {
+        return new Service[] { Service.DYNAMODB };
+    }
+
+    @Override
+    public void customize(Aws2TestEnvContext envContext) {
+
+        /* SQS */
```
Comment (aldettinger) [inline integration-tests-aws2/aws2-ddb/src/test/java/org/apache/camel/quarkus/component/aws2/ddb/it/Aws2DdbTestEnvCustomizer.java:47]: Same here

**Review by aldettinger — APPROVED** [review]
A bit of nitpicking :) LGTM. Nice to see the AWS2 test coverage growing :+1:

File: integration-tests-aws2/aws2-ddb/src/test/java/org/apache/camel/quarkus/component/aws2/ddb/it/Aws2DdbTest.java (hunk 1 above)
Comment (ppalaga) [reply integration-tests-aws2/aws2-ddb/src/test/java/org/apache/camel/quarkus/component/aws2/ddb/it/Aws2DdbTest.java]: Indeed, a copy and paste error :) Thanks for spotting it!

File: integration-tests-aws2/aws2-ddb/src/test/java/org/apache/camel/quarkus/component/aws2/ddb/it/Aws2DdbTestEnvCustomizer.java (hunk 2 above)
Comment (ppalaga) [reply integration-tests-aws2/aws2-ddb/src/test/java/org/apache/camel/quarkus/component/aws2/ddb/it/Aws2DdbTestEnvCustomizer.java]: ditto

**Comment by ppalaga** [comment]
058d403: removed the out-of-date sqs comments, fixed the generation of the grouped module

**Comment by ppalaga** [comment]
ignoring the spring-rabbitmq failure.

**Pull request merged**